### PR TITLE
Removed unneeded code generated by giter8 and added a test for Metric…

### PR DESCRIPTION
**Pull request by VincenzoMorenoLuna**
…sSettings

### CODE REVIEW

File: app/util/JsonFormatters.scala [hunk 1]
```diff
@@ -8,7 +8,7 @@ import scala.util.Try
 
 object JsonFormatters {
 
-  implicit val urlFormat = new Format[URL] {
+  implicit val urlFormat: Format[URL] = new Format[URL] {
     override def writes(url: URL): JsValue =
       JsString(url.toString)
```
Comment (danielwojdadepop) [inline app/util/JsonFormatters.scala]: Do we need that formatter?

File: app/util/JsonFormatters.scala [hunk 2]
```diff
@@ -22,7 +22,7 @@ object JsonFormatters {
     }
   }
 
-  implicit val mimeTypeFormat = new Format[MimeType] {
+  implicit val mimeTypeFormat: Format[MimeType] = new Format[MimeType] {
     override def writes(o: MimeType) = JsString(o.toString)
```
Comment (danielwojdadepop) [inline app/util/JsonFormatters.scala]: Do we need the MimeType Formatter?

File: build.sbt [hunk 3]
```diff
@@ -48,7 +48,7 @@ lazy val root = (project in file("."))
 lazy val playSettings =
   commonSettings ++ Seq(
     organization := "com.depop",
-    RoutesKeys.routesImport := Seq("binders.Implicits._", "models._"),
+    RoutesKeys.routesImport := Seq("models._"),
```
Comment (danielwojdadepop) [inline build.sbt]: I'd remove this line. PrevBanned will not have rest api.

File: app/util/JsonFormatters.scala (hunk 1 above)
Comment (VincenzoMorenoLuna) [reply app/util/JsonFormatters.scala]: Removed

File: app/util/JsonFormatters.scala (hunk 2 above)
Comment (VincenzoMorenoLuna) [reply app/util/JsonFormatters.scala]: Removed

File: build.sbt (hunk 3 above)
Comment (VincenzoMorenoLuna) [reply build.sbt]: Done

**Pull request merged**